### PR TITLE
update README, copyright

### PR DESCRIPTION
**Pull request by soniah**
Hi @SuperQ, I've updated the README - please let me know if you want any changes.

### CODE REVIEW

**Comment by soniah** [comment]
And @brian-brazil , sorry :-)

File: generic_e2e_test.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2012-2020 The GoSNMP Authors. All rights reserved.  Use of this
+// Copyright 2012-present The GoSNMP Authors. All rights reserved.  Use of this
```
Comment (SuperQ) [inline generic_e2e_test.go:1]: It's actually fine to [remove the end year completely](https://www.copyrightlaws.com/copyright-symbol-notice-year/#:~:text=The%20general%20rule%20is%20that,first%20publication%20of%20the%20work.) for this kind of notice. Only the initial copyright year is necessary.

```suggestion
// Copyright 2012 The GoSNMP Authors. All rights reserved.  Use of this
```

**Comment by SuperQ** [comment]
Minor suggestion for the copyright notice, otherwise LGTM.

**Comment by brian-brazil** [comment]
:+1:

**Comment by soniah** [comment]
Copyright updated all files, including examples.

**Review by SuperQ — APPROVED** [review]
LGTM

**Pull request merged**